kaldi/fstext/special.py

```diff
@@ -110,7 +110,7 @@ def determinize_lattice(ifst, compact_output=True,
     opts = _determinize_lattice.DeterminizeLatticeOptions()
     opts.delta, opts.max_mem, opts.max_loop = delta, max_mem, max_loop
     if compact_output:
-        ofst = _fst.CompactLatticeFst()
+        ofst = _fst.CompactLatticeVectorFst()
```

Comment:
It seems there are a bunch of similar errors in this module. Do you mind fixing them as well? The type of any output fst object should include "Vector" in its name, e.g. StdFst -> StdVectorFst.
 
We used to designate VectorFst types simply as Fst at the Python level to save some typing for the users. Later on, we introduced ConstFst types and renamed Fst to VectorFst to avoid any confusion but it seems we forgot to update this file to use the correct type names.